coalib/bearlib/aspects/Spelling.py

```diff
@@ -33,15 +33,39 @@ class docs:
         """
         example_language = 'English'
         importance_reason = """
-        Words should always be written as the are supposed to be.
+        Words should always be written as they are supposed to be.
         """
         fix_suggestions = """
         Use the correct spelling for the misspelled words, you simply use
-        a spellchecker.
+        a spell-checker.
         """
 
 
 @Spelling.subaspect
+class CompanySpecificMisSpelledWord:
```

Comment:
regarding casing: Misspelled is a word, not two. it should be CompanySpecificMispelledWord